app/controllers/offline_reservations_controller.rb

```diff
@@ -52,6 +53,12 @@ def update
 
   private
 
+  def flag_ongoing_reservations_as_problem
+    OrderDetail
+      .where(id: @instrument.reservations.ongoing.pluck(:order_detail_id))
```

Comment:
I'm not sure a callback is the right answer, but this seems a little too knowledgable about order details for a controller. Not sure I have a better solution, though.